doc/cephfs/index.rst

```diff
@@ -33,8 +33,19 @@ and Swift APIs, or native bindings (librados).
 Using CephFS
 ============
 
-Using the Ceph Filesystem requires at least one :term:`Ceph Metadata Server` in
-your Ceph Storage Cluster.
+Prerequisites for using CephFS with a running Ceph Storage Cluster:
+
+- At least one active :term:`Ceph Metadata Server`.
+- Minimum one data and metadata :term:`RADOS` pool.
+
+.. note::
+
+         If cephx authentication is enabled, secret key is required for
+         mounting CephFS. It is a file containing key extracted from keyring.
+
+         Using Erasure Coded(EC) pools with CephFS is supported only with
+         BlueStore Backend. They cannot be used as metadata pools and
+         overwrites should be enabled on the data pools.
```

Comment:
These details are distracting for a summary. Let's just make sure these notes are addressed in `client-auth.rst` and `createfs.rst`.

Reply:
Done.